src/librustc_typeck/diagnostics.rs

```diff
@@ -380,6 +380,24 @@ fn main() {
 ```
 "##,
 
+E0045: r##"
+Rust only supports variadic parameters for interoperability with C code in its
+FFI. As such, variadic parameters can only be used with functions which are
+using the C ABI. Examples of erroneous code:
+
+```
+extern "rust-call" { fn foo(x: u8, ...); }
+// or
+fn foo(x: u8, ...) {}
+```
+
+To fix such code, put them in extern "C" block:
```

Comment:
"put them in an extern ..."

Also, you misinterpreted my suggestion... there are two ways to fix this error, one is to do:

`extern "C" { ... }` and the other is to use `extern "C" fn`.